pkg/wolfssl/Makefile

```diff
@@ -0,0 +1,17 @@
+PKG_NAME=wolfssl
+PKG_URL=https://github.com/danielinux/wolfssl.git
+#PKG_URL=https://github.com/wolfssl/wolfssl.git
```

Comment:
is this meant to be changed to the upstream version for the final integration?

Reply:
Yes, and since wolfssl PR has been accepted last week, I've updated this so it finally points to the official masterbranch